### ISSUE TITLE
`position` servo parameters ?

### ISSUE THREAD

**Issue by Theo-Cheynel** [google/brax#333]
Hi,

I was wondering whether the `<position>` actuator was supported in v2, and whether we could change its gain parameters (in MJCF files, [the gains can be specified](https://mujoco.readthedocs.io/en/stable/XMLreference.html#actuator-motor:~:text=a%20position%20servo.-,The%20underlying%20general%20attributes%20are%20set%20as%20follows%3A,-Attribute)).

Thanks

**Comment by btaba** [comment]
Yes it's supported. We don't use gain, just gear, in a somewhat simplified version of the position actuator (still subject to change)
the torque for a position actuator is $gear * (ctrl - q)$

**Comment by Theo-Cheynel** [comment]
I see, thank you very much ! So there is only a proportional term, and no derivative term (w.r.t. $\dot{q}$) ?

**Comment by btaba** [comment]
the derivative term is added via damping of the joint
